NEWS.txt

```diff
@@ -3,249 +3,183 @@ Release Notes
-   - Better support for Windows end-lines into C codes
-   - Support for space in paths on Windows
+ - Scan improvements
 
- - Scan improvements:
+   - Speed up Theano scan compilation and gradient computation
+   - Added meaningful message when missing inputs to scan
 
-   - More scan optimizations, with faster compilation and gradient computation
-   - Support for checkpoint in scan (trade off between speed and memory usage, useful for long sequences)
-   - Fixed broadcast checking in scan
+ - Speed up graph toposort algorithm
+ - Faster compilation step by massively using a new interface for op params
+ - Faster optimization step
+ - Documentation updated and more complete
+ - Many bug fixes, crash fixes and warning improvements
 
- - Graphs improvements:
-
-   - More numerical stability by default for some graphs
-   - Better handling of corner cases for theano functions and graph optimizations
-   - More graph optimizations with faster compilation and execution
-   - smaller and more readable graph
-
- - New GPU back-end:
-
-   - Removed warp-synchronous programming to get good results with newer CUDA drivers
-   - More pooling support on GPU when cuDNN isn't available
-   - Full support of ignore_border option for pooling
-   - Inplace storage for shared variables
-   - float16 storage
-   - Using PCI bus ID of graphic cards for a better mapping between theano device number and nvidia-smi number
-   - Fixed offset error in ``GpuIncSubtensor``
-
- - Less C code compilation
- - Added support for bool dtype
- - Updated and more complete documentation
- - Bug fixes related to merge optimizer and shape inference
- - Lot of other bug fixes, crashes fixes and warning improvements
-
-A total of 123 people contributed to this release since 0.8.0, see list below.
+A total of 65 people contributed to this release since 0.9.0, see list below.
 
 Interface changes:
- - Merged ``CumsumOp/CumprodOp`` into ``CumOp``
- - In MRG module:
-
-   - Replaced method ``multinomial_wo_replacement()`` with new method ``choice()``
-   - Random generator now tries to infer the broadcast pattern of its output
-
- - New pooling interface
- - Pooling parameters can change at run time
- - Moved ``softsign`` out of sandbox to ``theano.tensor.nnet.softsign``
- - Using floatX dtype when converting empty list/tuple
- - ``Roll`` make the shift be modulo the size of the axis we roll on
- - ``round()`` default to the same as NumPy: half_to_even
+ - Changed ``grad()`` method to ``L_op()`` in ops that need the outputs to compute gradient
+ - Merged duplicated diagonal functions into two ops: ``ExtractDiag`` (extract a diagonal to a vector),
+   and ``AllocDiag`` (set a vector as a diagonal of an empty array)
+ - Replaced ``MultinomialWOReplacementFromUniform`` with ``ChoiceFromUniform``
+
+ - Removed or deprecated Theano flags:
+
+   - ``cublas.lib``
+   - ``cuda.enabled``
+   - ``enable_initial_driver_test``
+   - ``gpuarray.sync``
+   - ``home``
```

Comment:
I found it (`home`) in 0.9.X branch: https://github.com/Theano/Theano/blob/0.9.X/theano/configdefaults.py#L581

NB; But it is not used since a time.